crud/result.go

```diff
@@ -130,27 +132,27 @@ func (r *Result) DecodeMsgpack(d *msgpack.Decoder) error {
 		}
 	}
 
-	code, err := d.PeekCode()
-	if err != nil {
-		return err
-	}
-
-	var retErr error
-	if msgpackIsArray(code) {
-		crudErr := newErrorMany(r.rowType)
-		if err := d.Decode(&crudErr); err != nil {
-			return err
-		}
-		retErr = *crudErr
-	} else if code != msgpcode.Nil {
-		crudErr := newError(r.rowType)
-		if err := d.Decode(&crudErr); err != nil {
+	if arrLen > 1 {
```

Comment:
Please, update the method too:

https://github.com/tarantool/go-tarantool/blob/1f844f0c82aab458da6e28aa43bf9bf98677b25c/crud/result.go#L215

Reply:
Reworked methods to have the same structure